client-spark/spark-2/src/main/java/org/apache/spark/shuffle/celeborn/SparkShuffleManager.java

```diff
@@ -69,7 +69,7 @@ public SparkShuffleManager(SparkConf conf, boolean isDriver) {
     this.conf = conf;
     this.isDriver = isDriver;
     this.celebornConf = SparkUtils.fromSparkConf(conf);
-    this.cores = conf.getInt(SparkLauncher.EXECUTOR_CORES, 1);
+    this.cores = executorCores(conf);
```

Comment:
This change is blocked by https://github.com/apache/celeborn/pull/2562. cc @cxzl25.